grisera_api/time_series/time_series_service.py

```diff
@@ -0,0 +1,51 @@
+from graph_api_service import GraphApiService
+from time_series.time_series_model import TimeSeriesIn, TimeSeriesOut
+
+
+class TimeSeriesService:
+    """
+    Object to handle logic of time_series requests
+
+    Attributes:
+        graph_api_service (GraphApiService): Service used to communicate with Graph API
+    """
+    graph_api_service = GraphApiService()
+
+    def save_time_series(self, time_series: TimeSeriesIn):
+        """
+        Send request to graph api to create new time_series
+
+        Args:
+            time_series (TimeSeriesIn): TimeSeries to be added
+
+        Returns:
+            Result of request as time_series object
+        """
+        node_response_time_series = self.graph_api_service.create_node("TimeSeries")
+
+        if node_response_time_series["errors"] is not None:
+            return TimeSeriesOut(type=time_series.type, source=time_series.source,
+                                 observable_information_id=time_series.observable_information_id,
+                                 measure_id=time_series.measure_id,
+                                 additional_properties=time_series.additional_properties,
+                                 errors=node_response_time_series["errors"])
+
+        time_series_id = node_response_time_series["id"]
+        properties_response = self.graph_api_service.create_properties(time_series_id, time_series)
```

Comment:
When we create properties for time series, there are created properties like observable_information_id and measure_id ,what is a bug. To avoid that situation ids should be saved to other variables and set as None in time_series object.

Reply:
Already done 👌